Summary key error when user removes all summary key to plot.

While testing some recent changes I noticed the following error. 

```
Traceback (most recent call last):
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/dash/dash.py", line 1031, in dispatch
    response.set_data(func(*args, outputs_list=outputs_list))
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/dash/dash.py", line 966, in add_context
    output_value = func(*args, **kwargs)  # %% callback invoked %%
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/everviz/plugins/summary_plot/summary_plot.py", line 212, in update_graph
    data = calculate_statistics(data, key_list).set_index(
  File "/home/dan/miniconda/envs/everest/lib/python3.6/site-packages/everviz/plugins/summary_plot/util.py", line 9, in calculate_statistics
    raise AssertionError("List of summary keys is required")
AssertionError: List of summary keys is required
```

How to reproduce? 

Add more than one summary key to display in the summary keys plots and then remove them one by one, until there is no summary key selected.

--- a/everviz/plugins/summary_plot/summary_plot.py
+++ b/everviz/plugins/summary_plot/summary_plot.py
@@ -216,9 +216,7 @@ class SummaryPlot(EvervizPluginABC):
             # The key_list arguments is the list of keys to plot. The line_list
             # argument is a list of batches, or a list of dates to plot for
             # those keys.
-            if key_list is None or line_list is None:
-                return {}
-            if len(key_list) + len(line_list) <= 1:
+            if not key_list or not line_list:
                 return {}
 
             data = get_data(self.csv_file)

